Kickstarter-iOS/Views/Cells/SettingsCurrencyPickerCell.swift

```diff
@@ -54,7 +54,7 @@ extension SettingsCurrencyPickerCell: UIPickerViewDataSource {
 
 extension SettingsCurrencyPickerCell: UIPickerViewDelegate {
   func pickerView(_ pickerView: UIPickerView, titleForRow row: Int, forComponent component: Int) -> String? {
-    return Currency.allCases[row].descriptionText //Currency(rawValue: row)?.descriptionText
+    return Currency.allCases[row].descriptionText
```

Comment:
Good job killing this...I removed this on the `master` branch too :) 
![giphy](https://media.giphy.com/media/11b700y1OsNQcg/giphy.gif)